util/reggen/reg_pkg.sv.tpl

```diff
@@ -350,13 +350,15 @@ value = "{}'h {:x}".format(aw, r.offset)
 % endif
 </%def>\
 
-package ${lblock}_reg_pkg;
+package ${lblock}${alias_impl}_reg_pkg;
 % if localparams:
+% if not block.alias_impl:
```

Comment:
Nit: Maybe just `% if localparams and not block.alias_impl:` ?

Reply:
jep, let's simplify this.